Update CSS query classes doc (#59323)
woocommerce/woocommerce: docs/theming/block-theme-development/css-styling.md

}
```

### CSS Container Queries for Cart and Checkout blocks

> WooCommerce now defines the top-level wrappers of both the Cart and Checkout blocks as CSS containers using `container-type: inline-size`:

Reviewer: nit: "now" is unnecessary—document the present.

```suggestion
WooCommerce defines the top-level wrappers of Cart and Checkout blocks as CSS containers. These containers use `container-type: inline-size`:
```

```css
.wp-block-woocommerce-checkout {
	container-type: inline-size;
}
```

> Developers can now use CSS Container Queries (`@container`) to style child components based on parent width, eliminating the need for JS-added classes like `.is-large`. Legacy classes remain for compatibility, but we recommend using the new approach for cleaner styling and fewer layout shifts.

Reviewer: ```suggestion
Developers can use CSS Container Queries (`@container`) to style child components based on parent width, eliminating the need for JS-added classes such as `.is-large` which are considered legacy. Legacy classes remain in place for backwards compatibility, but we recommend using the container approach for cleaner styling and fewer layout shifts.
```

> **Note:** Container queries work as long as there isn't another container in between the checkout blocks wrapper and the element being styled. If you define additional CSS containers in your theme or plugins, they may interfere with the container query behavior.

Author: This part isn't mentioned in the developer advisory. Could you also include it there @bacoords ?

> We've introduced new SCSS mixins as convenient helpers for targeting the same width breakpoints previously defined by classes like `.is-large`, `.is-medium`, `.is-small` and `.is-mobile`. These mixins make it easier to style inner content within the Cart and Checkout blocks based on the container width.

Reviewer: ```suggestion
There are SCSS mixins available as convenient helpers for targeting the same width breakpoints previously defined by classes like `.is-large`, `.is-medium`, `.is-small` and `.is-mobile`. These mixins make it easier to style inner content within the Cart and Checkout blocks based on the container width.
```
